src/plugins/console/public/application/components/settings/settings_editor.tsx

```diff
@@ -337,10 +337,6 @@ export const SettingsEditor = (props: Props) => {
             label={i18n.translate('console.settingsPage.refreshingDataLabel', {
               defaultMessage: 'Refresh frequency',
             })}
-            labelWarning={i18n.translate('console.settingsPage.refreshingDataWarningLabel', {
```

Comment:
We had this copy written twice, here in the tooltip and in the formgroup right above.